### ISSUE TITLE
How do you change the position of the color ramp?

### ISSUE THREAD

**Issue by sam-mahdi** [schrodinger/pymol-open-source#308]
When using cmd.ramp_new(), an object (a color bar) is generated that is located at the bottom center of the screen. How is this moved however? E.G. Have the color bar be on the center left, or center right? Something like this

![image](https://github.com/schrodinger/pymol-open-source/assets/65124322/6b947633-a2d8-4f31-b05d-7988d5367063)

**Comment by JarrettSJohnson** [comment]
From what I see, this is all hardcoded, but it's definitely reasonable to have this be moved and rotated around to fit your use-cases.

**Comment by sam-mahdi** [comment]
Hi Jarrett, 

Are you saying that there are no arguments built into the color bar allowing it to move around?

**Comment by JarrettSJohnson** [comment]
Correct. AFAIK, this is not yet possible.

**Comment by sam-mahdi** [comment]
Do I close this then (I'm not very familiar with how github issues work, since my issue was answered)?

**Comment by JarrettSJohnson** [comment]
Let's leave it up as as a feature request. I do think something like this is helpful to have in the future.

**Comment by varshneydevansh** [comment]
Is my understanding correct

Currently, in PyMOL, the cmd.ramp_new() function does not have built-in arguments or methods to directly adjust the position of the color bar on the screen. The color bar generated by cmd.ramp_new() is hardcoded to appear at the bottom center of the screen. 

The current implementation does not support moving it to different screen locations, such as center left or center-right.

**Comment by JarrettSJohnson** [comment]
Right. From what I see, the positioning and orientation is pretty much hardcoded. The way I'd approach this is to allow `transform_object` (or similar command) to alter the `ObjectGadgetRamp` transform (or position/rotation).

**Comment by varshneydevansh** [comment]
Okay. I'll try to get this done :)

I came across to this project as my friend is using this tool for his master’s thesis on PEMFC.
